client/utils.c

```diff
@@ -764,3 +764,48 @@ TDNFReadAutoInstalled(
     TDNF_SAFE_FREE_STRINGARRAY(ppszAutoInstalled);
     goto cleanup;
 }
+
+void
+TDNFFreeHistoryInfoItems(
+    PTDNF_HISTORY_INFO_ITEM pHistoryItems,
+    int nCount
+)
+{
+    if (pHistoryItems)
```

Comment:
We can put it like args check as in TDNF all places it will maintain the same decorum

Reply:
We don't do that for the `TDNFree*` functions because they return `void`.